recipe/conda_build_config.yaml

```diff
@@ -1,9 +0,0 @@
-mkl_support: # [linux64 or win64]
```

Comment:
@sanurielf, why was it removed? As far as I can tell, tthere is no mkl build on linux anymore and this configuration is required to build with/without mkl. @ocefpaf does it make sense?

Reply:
It was removed by conda smithy rerender. Tests are passing on Linux.

Reply:
Thank you @sanurielf for the swift reply. This sounds surprising and I will give a quick go at it to try to add it back.